packages/math-input/src/components/keypad-container.tsx

```diff
@@ -301,6 +358,16 @@ const mapStateToProps = (state: ReduxState): ReduxProps => {
         active: state.keypad.active,
         layoutMode: state.layout.layoutMode,
         navigationPadEnabled: state.layout.navigationPadEnabled,
+        currentPage: state.pager.currentPage,
+        cursorContext: state.input.cursor?.context,
+        dynamicJumpOut: !state.layout.navigationPadEnabled,
+        paginationEnabled: state.layout.paginationEnabled,
+        echoes: state.echoes.echoes,
+        popover: state.gestures.popover,
+        gestureManager: state.gestures.gestureManager,
+        gestureFocus: state.gestures.focus,
+        heightPx: state.layout.buttonDimensions.heightPx,
+        widthPx: state.layout.buttonDimensions.widthPx,
```

Comment:
**suggestion:** instead of passing this around as separate values everywhere, maybe combine these into a data structure, e.g. `Dimensions` that can be passed around.